label_studio/users/api.py

```diff
@@ -75,16 +75,16 @@ def create(self, request, *args, **kwargs):
     @swagger_auto_schema(
         tags=['Users'],
         operation_summary='Get user info',
-        operation_description='Get info about a specific Label Studio user.'
+        operation_description='Get info about a specific Label Studio user, based on the user ID.'
     )
     def retrieve(self, request, *args, **kwargs):
         return super(UserAPI, self).retrieve(request, *args, **kwargs)
 
     @swagger_auto_schema(
         tags=['Users'],
         operation_summary='Update user details',
```

Comment:
As an administrator, I get an error that I don't have permissions to make these changes on a different user, so then who does have permissions?

Reply:
This was using the enterprise API instead of open source but might still be an issue?